lib/prawnto/compile_support.rb

```diff
@@ -65,7 +65,7 @@ def set_cache_control
     end
 
     def set_content_type
-      @controller.response.content_type ||= Mime[:pdf]
+      @controller.response.content_type = Mime[:pdf]
```

Comment:
@TJKruitbosch this is a functionality change - why did this one change?

Reply:
The deprecation warning appearing on this line is `DEPRECATION WARNING: Rails 6.1 will return Content-Type header without modification. If you want just the MIME type, please use '#media_type' instead. (called from set_content_type at <workspace>/prawnto/lib/prawnto/compile_support.rb:68)`

I took a quick peek at the [Rails code](https://github.com/rails/rails/blob/fbe2433be6e052a1acac63c7faf287c52ed3c5ba/actionpack/lib/action_dispatch/http/response.rb#L235) this is calling to understand more about the change, and it looked like the warning was specifically about reading `content_type` rather than setting it, so that's just the part we need to solve.

If I use the "media_type" as suggested in the warning (something like `@controller.response.content_type = @controller.response.media_type || Mime[:pdf]`), it also passes specs and basic QAing, but at first glance I wasn't entirely sure they contractually return the same thing. When writing this response I took a second look and think that it might actually be the correct solution though...thoughts?

Reply:
Thanks again for having me take a second look at this @jessevanherk. Backed out the change, and decided on using the new config flag for the dummy spec install (`config.action_dispatch.return_only_media_type_on_content_type = false` in spec/dummy/config/application.rb). Shouldn't be a functionality change in the gem anymore, but the deprecation warning will still appear for apps using this gem until they either use that flag as well or upgrade to Rails 6.1+